pkgs/applications/emulators/c64-debugger/default.nix

```
@@ -0,0 +1,90 @@
+{ fetchgit, lib, pkgs, stdenv }:
```

```suggestion
{ lib
, stdenv
, fetchgit
}:
```

Never list `pkgs` as input.